hera_mc/weather.py

```diff
@@ -25,19 +25,19 @@
 #   https://github.com/ska-sa/katportalclient/tree/master/examples
 weather_sensor_dict = {'wind_speed': {'sensor_name': 'anc_mean_wind_speed',
                                       'units': 'm/s',
-                                      'reduction': 'mean', 'period': 60,
+                                      'reduction': 'mean', 'period': 300,
```

Comment:
Really, please, separate unrelated changes into different pull requests. The git/GitHub history is a valuable asset for understanding the codebase and part of that value comes from keeping the intellectual content of the code well-matched to the concepts used by our change management tools.